core/src/database/ProxyBackend.cpp

```diff
@@ -0,0 +1,55 @@
+/*
+ *
+ * ProxyBackend.cpp
+ * ledger-core
+ *
+ * Created by Pierre Pollastri on 13/11/2018.
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017 Ledger
```

Comment:
2018 I guess :P

Reply:
better be prepared for 2019

Reply:
I'll stay in 2017 until Bitcoin rises and shines again.